From 49dddc668ec2020529d8248985d7833de2eed7f3 Mon Sep 17 00:00:00 2001
From: Tobias Ruck <ruck.tobias@gmail.com>
Date: Wed, 24 Sep 2025 20:29:28 +0200
Subject: [PATCH] [doged] Fix chronik_electrum_blockchain.py

This test expects the larger OP_RETURN size standardness rule, so we add `-datacarriersize=223`

(cherry picked from commit 2fde7d8a4039f3368e1ff87912354619ff04b862)

diff --git a/test/functional/chronik_electrum_blockchain.py b/test/functional/chronik_electrum_blockchain.py
index a7cd5dad301a339813c8ccd2b29f7bb8dcbf18d7..dcd0669ba2026cef9cc733875f4f11bec591b032 100644
--- a/test/functional/chronik_electrum_blockchain.py
+++ b/test/functional/chronik_electrum_blockchain.py
@@ -63,6 +63,7 @@ class ChronikElectrumBlockchain(BitcoinTestFramework):
             [
                 "-chronik",
                 f"-chronikelectrumbind=127.0.0.1:{chronikelectrum_port(0)}:t",
+                "-datacarriersize=223",
             ]
         ]
 
